README.md

```diff
@@ -23,7 +23,17 @@
 ----
 
 ## Descrizione del progetto
-TBD
+Il progetto IO:
+- è l'app dei servizi pubblici;
+- è il canale per facilitare la comunicazione fra i cittadini e lo Stato;
+- promuove una nuova generazione di Servizi Pubblici, concepiti e progettati insieme ai cittadini a partire dai loro bisogni;
+- è un vettore attraverso il quale le istituzioni governative veicolano iniziative sociali/economiche/sanitarie;
```

Comment:
Just a minor style proposta: use commas instead of slashes as it reads more fluently.
```suggestion
- è un vettore attraverso il quale le istituzioni governative veicolano iniziative sociali, economiche e sanitarie;
```